core/src/test/java/io/questdb/test/griffin/CheckpointTest.java

```diff
@@ -49,11 +49,12 @@
 
 import static io.questdb.PropertyKey.CAIRO_SNAPSHOT_RECOVERY_ENABLED;
 
-public class SnapshotTest extends AbstractCairoTest {
+public class CheckpointTest extends AbstractCairoTest {
```

Comment:
There are no tests here that would call `snapshot complete;`, so backward compatibility may be broken in the future without us noticing.

Reply:
added exercising of legacy commands to checkpoint fuzz test here https://github.com/questdb/questdb/pull/4821/commits/54be271306f73d2a04f6caf59638838b561fbaa1